config/context/spring-context.xml

```diff
@@ -126,9 +126,29 @@
     <constructor-arg name="keyPair" ref="brvsAccountKeyPair"/>
     <constructor-arg name="irohaAPI" ref="irohaAPI"/>
   </bean>
+  <bean id="updateWhitelistRule" class="iroha.validation.rules.impl.whitelist.UpdateWhitelistRule">
+    <constructor-arg name="brvsAccountId" value="${credential.accountId}"/>
+    <constructor-arg name="brvsAccountKeyPair" ref="brvsAccountKeyPair"/>
+    <constructor-arg name="irohaAPI" ref="irohaAPI"/>
+    <constructor-arg name="validationPeriod" value="${whitelist.validation}"/>
+  </bean>
+  <bean id="checkEthWhitelistRule" class="iroha.validation.rules.impl.whitelist.CheckWhitelistRule">
+    <constructor-arg name="brvsAccountId" value="${credential.accountId}"/>
+    <constructor-arg name="brvsAccountKeyPair" ref="brvsAccountKeyPair"/>
+    <constructor-arg name="irohaAPI" ref="irohaAPI"/>
```

Comment:
Don't you forget to add `withdrawalAccount`?

Reply:
yep, fixed